client/components/domains/register-domain-step/index.jsx

```diff
@@ -783,12 +785,10 @@ class RegisterDomainStep extends React.Component {
 	};
 
 	getSubdomainSuggestions = ( domain, timestamp ) => {
-		const includeWordPressDotCom =
-			this.props.surveyVertical && this.props.includeDotBlogSubdomain ? false : true;
 		const subdomainQuery = {
 			query: domain,
 			quantity: 1,
-			include_wordpressdotcom: includeWordPressDotCom,
+			include_wordpressdotcom: ! this.props.includeDotBlogSubdomain,
```

Comment:
If one of those is true, it will be loading results. It's possible that someone does introduce errors, but before it wasn't even checking the `includeDotBlogSubdomain` at all. 😆